### PR TITLE
Devise

### PR DESCRIPTION
**Pull request by ezeqzim**
## Summary
Added User Authentication using Devise.

## Test Plan
Sign up, log in, log out some users.

## Trello Card
https://trello.com/c/gYiWECr7/10-instalar-y-configurar-devise

## Notes
Devise added lots of files I didn't change them.
Don't know why ALL the files look like new ones... The ones from blog-base remain the same...

### CODE REVIEW

**Comment by NicoBuch** [comment]
You rebased from something that its not in development. So, this pull request should have as target that branch.

**Comment by ezeqzim** [comment]
Now blog base is merged with master, remember most of the files were generated by devise gem

File: blog/app/views/welcome/index.html.slim [hunk 1]
```diff
@@ -1,2 +1,7 @@
 h1 Hello, Rails!
-= link_to 'My Blog', controller: 'articles'
+- if user_signed_in?
+  = link_to 'My Blog', controller: 'articles'
+  br
+  = link_to 'Sign out', destroy_user_session_path, :method => 'delete'
```
Comment (NicoBuch) [inline blog/app/views/welcome/index.html.slim]: Use the Ruby new hash syntax: `method: :delete`
The new hash syntax is replacing `:something =>` for `something:`

**Comment by NicoBuch** [comment]
The PR its almost okey. Check what I marked you there about the ruby syntax. From now on, always write it like that.
Delete test files generated by devise. 
Check what happens if you delete all of the devise controllers. (The entire users folder) If it still works, better, if not, go back and leave it just as it is.
Then squash and merge :+1:

**Pull request merged**